x-pack/plugins/task_manager/server/polling/task_poller.ts

```diff
@@ -128,10 +129,12 @@ function asPollingError<T>(err: string | Error, type: PollingErrorType, data: Op
 export class PollingError<T> extends Error {
   public readonly type: PollingErrorType;
   public readonly data: Option<T>;
+  public readonly source: TaskErrorSource;
```

Comment:
what is this used for in this PR?

Reply:
TM collects metrics only from `TaskRun` or `TaskPollingCycle` events and polling errors are always framework error.
We decided to show it explicitly.